src/components/table/OcTableFiles.vue

```diff
@@ -626,6 +648,10 @@ export default {
       }
     },
     methods: {
+      fileDropped(fileId) {
+        var selectedString = this.selectedIds.join(`, `)
```

Comment:
Generally I would like to stay away from `var` if possible and use `const` & `let` instead

Reply:
yes will try to remember it its a bad habit of mine :D